src/libasr/ASR.asdl

```diff
@@ -190,7 +190,7 @@ stmt
               expr? read, expr? write, expr? readwrite, expr? delim,
               expr? pad, expr? flen, expr? blocksize, expr? convert,
               expr? carriagecontrol, expr? iolength)
-    | FileWrite(int label, expr? unit, expr? fmt, expr? iomsg, expr? iostat, expr? id, expr* values)
+    | FileWrite(int label, expr? unit, expr? fmt, expr? iomsg, expr? iostat, expr? id, expr* values, expr? separator)
```

Comment:
Since we are not hooking this up in this PR, I would not add it to `FileWrite` yet. I am hoping to figure out a general solution for file IO that works well for both LPython and LFortran.

Reply:
I need to make that change because `handle_print` is also used by `FileWrite`.

Reply:
I see, keep it there then.